chainntnfs/neutrinonotify/neutrino.go

```diff
@@ -810,11 +831,11 @@ func (n *NeutrinoNotifier) RegisterSpendNtfn(outpoint *wire.OutPoint,
 	}
 
 	// Ensure that neutrino is caught up to the height hint before we
-	// attempt to fetch the utxo fromt the chain. If we're behind, then we
+	// attempt to fetch the utxo from the chain. If we're behind, then we
 	// may miss a notification dispatch.
 	for {
 		n.heightMtx.RLock()
-		currentHeight = n.bestHeight
+		currentHeight := n.bestHeight
```

Comment:
wasn't this fixed already? Might need a rebase.

Reply:
fixed

Reply:
This was just change here: https://github.com/lightningnetwork/lnd/pull/1953

Why change it back?